### PR TITLE
feat: add redirect rules

### PR DESCRIPTION
**Pull request by alexanderleegs**

### CODE REVIEW

**Comment by QiluXie** [comment]
@alexanderleegs does this need to be merged?

File: rebuild-sites-amplify/change-redirs.js
Comment (seaerchin) [inline rebuild-sites-amplify/change-redirs.js:1]: can we add a readme? that'd be useful for the env vars especially

File: rebuild-sites-amplify/change-redirs.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+});
+
+async function changeRedirs() {
+  let sitesList = {}
+  let nextToken
+  const listCommand = new ListAppsCommand({
+    maxResults: MAX_RESULTS_PER_PAGE,
+  })
+  response = await (awsClient.send(listCommand))
```
Comment (seaerchin) [inline rebuild-sites-amplify/change-redirs.js:21]: are we not missing a `let` or a `const` here

File: rebuild-sites-amplify/change-redirs.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    })
+    response = await (awsClient.send(listCommand))
+    nextToken = response.nextToken
+  }
+  for (const site in sitesList) {
+    const siteInfo = sitesList[site]
+    let alreadyHasRule = false
+    siteInfo.customRules.forEach(rule => {
+      if (rule.source === "</%5c/>") alreadyHasRule = true
+      else if (rule.source === "</%5C/>") alreadyHasRule = true
```
Comment (seaerchin) [inline rebuild-sites-amplify/change-redirs.js:45]: supernit: actually can just `.toLowerCase` but this is clear too

Comment (seaerchin) [reply rebuild-sites-amplify/change-redirs.js:45]: also we can use `.some` here

**Review by seaerchin — APPROVED** [review]
lgtm once docs added

File: rebuild-sites-amplify/change-redirs.js
Comment (alexanderleegs) [reply rebuild-sites-amplify/change-redirs.js:1]: Added in https://github.com/isomerpages/isomer-tooling/pull/44/commits/617ff6926ae5a1c648c01c6842143bb7a6e54b9c!

File: rebuild-sites-amplify/change-redirs.js (hunk 1 above)
Comment (alexanderleegs) [reply rebuild-sites-amplify/change-redirs.js]: Huh you're right, not sure how it was working - fixed in https://github.com/isomerpages/isomer-tooling/pull/44/commits/446ebd92845fb56a9f69b0cdb31fa412f68dee5d

File: rebuild-sites-amplify/change-redirs.js (hunk 2 above)
Comment (alexanderleegs) [reply rebuild-sites-amplify/change-redirs.js]: Swapped to check only for lowercase https://github.com/isomerpages/isomer-tooling/pull/44/commits/446ebd92845fb56a9f69b0cdb31fa412f68dee5d

**Pull request merged**